react-app/ccf/src/functions/storage.ts

```diff
@@ -0,0 +1,39 @@
+import { ref, uploadBytes, getDownloadURL, listAll } from "firebase/storage";
+import { storage } from '../index'
+
+export const handleUpload = async (file: File): Promise<string> => {
```

Comment:
I think the name of this function can be more descriptive. Something like "uploadFileToStorage" would be more clear about what the function does and will make code elsewhere easier to read.